lib/pointTransformer.js

```diff
@@ -5,6 +5,64 @@ var formatAddress = require('../lib/formatAddress');
 
 var loadDate = Date.now();
 
+var lats = [
+  'y',
+  'latitude',
+  'lat',
+  'lat_dd',
+  'lat_y'
+  ];
+
+var lons = [
+  'x',
+  'longitude',
+  'long',
+  'lon',
+  'lon_dd',
+  'long_dd',
+  'lon_x',
+  'long_x'
+];
+
+
+function getGeometry(props, geo){
+  if(geo !== null) return geo;
+
+  if(this.coordinates.x){
+    return {
+      "type":"Point",
+      "coordinates":[+props[this.coordinates.x], +props[this.coordinates.y]]
+    }
+  }
+
+  var lowerCase = {};
+  Object.keys(props).forEach(function(v){
+    lowerCase[v.toLowerCase()] = {key: v, val: props[v]};
+  });
+
+  var xMatch = matchFromArr(lowerCase, lons);
+  var yMatch = matchFromArr(lowerCase, lats);
+  
+  this.coordinates.x = xMatch.key;
+  this.coordinates.y = yMatch.key
```

Comment:
So I have required semi-colons turned off in my linter and am pretty agnostic about putting them after every statement. That said, the consistency is probably warranted.